nixos/modules/tasks/network-interfaces.nix

```
@@ -996,6 +997,64 @@ in
       });
     };
 
+    networking.gre = mkOption {
```

in keeping with other options this should be pluralized, but `gres` seems more confusing than helpful at first glance. how about
```suggestion
    networking.greTunnels = mkOption {
```